src/climate_learn/models/hub/interpolation.py

```diff
@@ -0,0 +1,13 @@
+# Third party
+from torch import nn
+import torch.nn.functional as F
+
+
+class Interpolation(nn.Module):
```

Comment:
I think the classe should be decorated using `register` (`register("interpolation"`)).